app/controllers/tags_controller.rb

```diff
@@ -134,10 +134,13 @@ def feed
   def show_hidden
     unless params[:creation_id].blank? || params[:creation_type].blank? || params[:tag_type].blank?
       raise "Redshirt: Attempted to constantize invalid class initialize show_hidden #{params[:creation_type].classify}" unless %w(Series Work Chapter).include?(params[:creation_type].classify)
-      model = begin
-                params[:creation_type].classify.constantize
-              rescue
-                nil
+      model = case params[:creation_type].downcase
```

Comment:
Definitely an improvement over `rescue nil` 😖 
I assume this will set the model to nil anyway if the creation_type didn't match.

Reply:
yes